packages/integrations/mdx/src/index.ts

```diff
@@ -156,9 +152,19 @@ export default function mdx(mdxOptions: MdxOptions = {}): AstroIntegration {
 										)}) };`;
 									}
 									if (!moduleExports.includes('Content')) {
-										code += `\nexport const Content = MDXContent;`;
+										// Make `Content` the default export so we can wrap `MDXContent` and pass in `Fragment`
+										code = code.replace('export default MDXContent;', '');
+										code += `\nexport const Content = (props = {}) => MDXContent({
+											...props,
+											components: { Fragment, ...props.components },
```

Comment:
In my tests, compiled MDX already includes an unused `Fragment` import, so using that here.

`parseESM` does return a list of imports too, so we could potentially use that to be extra sure the import exists if we think there are circumstances where it might not?

Reply:
Looks like it's coming from https://github.com/withastro/astro/blob/a1885ea2f59f26cbdae10c298e1e0d1063d9dca1/packages/astro/src/jsx/babel.ts#L150-L157. I think checking from `parseESM` would be great since it's free and gives better errors in case we miss it. 👍